scss/_root.scss

```diff
@@ -169,6 +170,7 @@
     --#{$prefix}link-hover-color: #{$link-hover-color-dark};
     --#{$prefix}link-color-rgb: #{to-rgb($link-color-dark)};
     --#{$prefix}link-hover-color-rgb: #{to-rgb($link-hover-color-dark)};
+    --#{$prefix}btn-link-focus-shadow-rgb: #{$btn-link-focus-shadow-rgb-dark};
```

Comment:
TBH, I'm not a big fan adding a component CSS variable in the `:root` since it will increase the size of all our bundles (all don't have buttons imported inside).